pkg/server/plugin/notifier/k8sbundle/k8sbundle.go

```diff
@@ -192,7 +192,7 @@ func (p *Plugin) setConfig(config *pluginConfig) error {
 		p.cancelWatcher()
 		p.cancelWatcher = nil
 	}
-	if config.WebhookLabel != "" {
+	if config.WebhookLabel != "" || config.APIServiceLabel != "" {
```

Comment:
Super curious how you noticed this, as SPIRE itself currently only configures once...?

Reply:
I was making the change to add that new flag and noticed this bit was missing.